libs/designsystem/modal/src/modal/services/alert.helper.ts

```diff
@@ -26,27 +27,27 @@ export class AlertHelper {
     };
   }
 
-  private getComponentProps(config: AlertConfig) {
+  public getComponentProps(config: AlertConfig) {
```

Comment:
Is this purely for testing purposes?
Then consider accessing the private function in the test suite instead `const props = alertHelper['getComponentProps'](config);`.